app/tests/unit/services/object.spec.js

```diff
@@ -0,0 +1,100 @@
+const { MockModel, MockTransaction } = require('../../common/dbHelper');
+
+jest.mock('../../../src/db/models/tables/objectModel', () => MockModel);
+
+const service = require('../../../src/services/object');
+
+const bucketId = '00000000-0000-0000-0000-000000000000';
+const objectId = '00000000-0000-0000-0000-000000000000';
+const userId = '00000000-0000-0000-0000-000000000000';
+
+const data = {
+  id: objectId,
+  bucketId: bucketId,
+  path: 'path',
+  public: 'true',
+  active: 'true',
+  createdBy: userId
+};
+
+const params = {
+  id: objectId,
+  bucketId: bucketId,
+  path: 'path',
+  public: 'true',
+  active: 'true',
+  createdBy: userId,
+  mimeType: 'mime',
+  deleteMarker: 'delete',
+  latest: 'latest'
+};
+
+beforeEach(() => {
+  MockModel.mockReset();
+  MockTransaction.mockReset();
+});
+
+afterEach(() => {
+  jest.clearAllMocks();
+});
+
+describe('bucket', () => {
+  const createSpy = jest.spyOn(service, 'create');
```

Comment:
Because we want to test the service calls. we dont actually want to mock these.
other wise they will never run. see comment below